projects/kit/src/lib/masks/number/number-mask.ts

```diff
@@ -43,6 +43,7 @@ export function maskitoNumberOptionsGenerator({
     decimalZeroPadding = false,
     prefix: unsafePrefix = '',
     postfix = '',
+    minusSign = CHAR_MINUS,
```

Comment:
You should also update this line
https://github.com/taiga-family/maskito/blob/b6cd193d7abe4d54c7b6c2a943ef72a682eeaf77/projects/kit/src/lib/masks/number/processors/min-max-postprocessor.ts#L37

\+ add cypress test for this case